web/templates/summary.html

```diff
@@ -0,0 +1,255 @@
+{% load static %}
+<html>
+    <head>
+        <style>
+            @font-face {
+            font-family: 'Marianne';
+            src: url({% static '../../static/fonts/Marianne-Regular.ttf' %}) format('truetype');
+            }
+            @font-face {
+                font-family: 'Marianne-Medium';
+                src:  url({% static '../../static/fonts/Marianne-Medium.ttf' %}) format('truetype');
+            }
+            @font-face {
+                font-family: 'Marianne-Bold';
+                src:  url({% static '../../static/fonts/Marianne-Bold.ttf' %}) format('truetype');
+            }
+            @font-face {
+                font-family: 'Marianne-ExtraBold';
+                src: url({% static '../../static/fonts/Marianne-ExtraBold.ttf' %}) format('truetype');
+            }
+            body {
+                font-family: 'Marianne', Arial, Helvetica, sans-serif;
+            }
+            @page {
+                size: A4 portrait;
+                margin: 1.3cm 2cm;
+
+                @frame footer_frame {
+                    -pdf-frame-content: footer_content;
+                    left: 500pt; width: 512pt; top: 820pt; height: 20pt;
+                }
+            }
+        </style>
+    </head>
+    <body style="font-family: 'Marianne';">
+        <div id="footer_content">Page <pdf:pagenumber>
+            de <pdf:pagecount>
+        </div>
+        <table>
+            <tr>
+                <td>
+                    <img width="100" src="{% static '../../static/images/certificate/masa.png' %}" alt="">
+                </td>
+                <td align="right">
+                    <img width="120" src="{% static '../../static/images/logo.png' %}" alt="">
+                </td>
+            </tr>
+        </table>
+        <div style="margin-top: 20px;">
+            <h1>Votre déclaration « {{ declaration.name }} »</h1>
+            {% if submission_date %}
+            <p>Soumise le {{ submission_date|date:"l j F Y" }}</p>
+            {% endif %}
+
+            <!-- Date de la déclaration -->
```

Comment:
comment out of place

Reply:
Fix : https://github.com/betagouv/complements-alimentaires/pull/1418/commits/c34685f2f6a7233c61c9df38ae517b4096b5fa30